diff --git a/rpxy-acme/Cargo.toml b/rpxy-acme/Cargo.toml
index f1d31cf..4a93685 100644
--- a/rpxy-acme/Cargo.toml
+++ b/rpxy-acme/Cargo.toml
@@ -20,7 +20,7 @@ thiserror = "2.0.11"
 tracing = "0.1.41"
 async-trait = "0.1.86"
 base64 = "0.22.1"
-aws-lc-rs = { version = "1.12.3", default-features = false, features = [
+aws-lc-rs = { version = "1.12.2", default-features = false, features = [
   "aws-lc-sys",
 ] }
 blocking = "1.6.1"
